cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -293,27 +297,38 @@ metadata:
     operator: rook
     storage-backend: ceph
 rules:
-  - apiGroups:
-      - ""
+  - apiGroups: [""]
+    resources: ["secrets", "configmaps"]
     verbs:
-      - "*"
-    resources:
-      - secrets
-      - configmaps
-  - apiGroups:
-      - storage.k8s.io
-    resources:
-      - storageclasses
+      # OBC controller creates secrets and configmaps containing information for users about how to
+      # connect to object buckets. It deletes them when an OBC is deleted.
+      - get
+      - create
+      - delete
```

Comment:
It's strange to see they aren't they garbaged collected once the OBC is deleted.

Reply:
l-b-p is very much still a POC library. They have finalizers and are deleted manually. I don't think they have an owner ref, but I could be mistaken. Either way, the lib calls `delete`.